lib/verifly/dependent_callbacks/invoker.rb

```diff
@@ -135,14 +137,19 @@ def log!(severity, message, callback: nil)
         DependentCallbacks.logger.public_send(severity, "Verifly::DependentCallbacks::Invoker") do
           if callback
             <<~TXT.squish if callback
-              #{message} at #{callback.name || "(anonymous)"}
+              #{message} callback #{callback.name || "(anonymous)"}
                          in #{callback.action.source_location(binding_)&.join(':')}
             TXT
           else
             message
           end
         end
       end
+
+      def call_with_time_report!(callback, *args) # :nodoc:
+        time_in_ms = Benchmark.realtime { callback.call(*args) } * 1000
```

Comment:
you should use something like `return callback.call(*args) unless DependentCallbacks.logger.info?` for better performance